plexpy/__init__.py

```diff
@@ -429,6 +429,7 @@ def initialize_scheduler():
 
         # Update check
         github_minutes = CONFIG.CHECK_GITHUB_INTERVAL if CONFIG.CHECK_GITHUB_INTERVAL and CONFIG.CHECK_GITHUB else 0
+        pms_update_notify_hours = CONFIG.PMS_UPDATE_NOTIFY_INTERVAL if 1 <= CONFIG.PMS_UPDATE_NOTIFY_INTERVAL <= 999 else 24
```

Comment:
Rename all `PMS_UPDATE_NOTIFY_INTERVAL` to `PMS_UPDATE_CHECK_INTERVAL`.

Only minimum of 1 hour.

```suggestion
        pms_update_check_hours = CONFIG.PMS_UPDATE_CHECK_INTERVAL if 1 <= CONFIG.PMS_UPDATE_CHECK_INTERVAL else 24
```